GREEDY/README.md

```diff
@@ -0,0 +1,59 @@
+# barcode-layout
+
+This repository provides a CUDA-optimized implementation of the GREEDY algorithm to optimize the layout of DNA-barcodes on 2d arrays.
+
+## Dependencies
+
+You need the follwing dependencies present on your system.
+
+- CUDA 11.x with `nvcc` in the search path
+
+## Building the program
+
+The algorithm can be compiled with different parameters:
+
+- NUMBER_OF_BARCODES: This parameter specifies the number of newline separated barcodes to read from stdin.
+- BARCODE_LENGTH: This specifies the length of each barcode in bytes.
+- ROW_COUNT and COL_COUNT: These specifiy the dimensions of the array on which the barcodes should be layed out. Note that `(ROW_COUNT * COL_COUNT) <= NUMBER_OF_BARCODES` must be true.
+- NEIGHBORHOOD: This describes the neighborhood strategy to apply. A default strategy which uses 8 unweighted neighbors is provided.
+
+Now you can build the program using
+
+```
+./scripts/build_greedy.sh <NUMBER_OF_BARCODES> <BARCODE_LENGTH> <ROW_COUNT> <COL_COUNT> [ <NEIGHBORHOOD_NAME> ]
```

Comment:
Das Skript build_greedy.sh existiert nicht, vermutlich ist build.sh gemeint (?)
Das Skript muss zuerst ausführbar gemacht werden, wollen wir das in der Installationsanleitung hinzufügen?

Reply:
erledigt

Reply:
Der Algorithmus ist in seiner aktuellen Form nun auch nochmal von mir getestet worden und außerdem auch auf Windows lauffähig.
Ich habe außerdem noch einiges in Go geschriebenes Tooling aus meiner BA übernommen und aufbereitet.